src/coreclr/src/System.Private.CoreLib/src/System/Math.CoreCLR.cs

```diff
@@ -97,9 +97,6 @@ public static partial class Math
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern double Pow(double x, double y);
 
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        public static extern double ScaleB(double x, int n);
```

Comment:
Could you please delete it from the unmanaged runtime:

src/coreclr/src/classlibnative/float/floatdouble.cpp
src/coreclr/src/classlibnative/float/floatsingle.cpp
src/coreclr/src/classlibnative/inc/floatdouble.h
src/coreclr/src/classlibnative/inc/floatsingle.h
src/coreclr/src/vm/ecalllist.h
Everything that references scalbn and scalbnf under src/coreclr/src/pal

Reply:
Also, I think these lines can be deleted from mono: https://github.com/dotnet/runtime/blob/2a1735b162437bb35a112014092a834c2496c3e3/src/mono/mono/metadata/sysmath.c#L221-L225 https://github.com/dotnet/runtime/blob/2a1735b162437bb35a112014092a834c2496c3e3/src/mono/mono/metadata/sysmath.c#L390-L394 https://github.com/dotnet/runtime/blob/2a1735b162437bb35a112014092a834c2496c3e3/src/mono/mono/metadata/icall-def-netcore.h#L148 https://github.com/dotnet/runtime/blob/2a1735b162437bb35a112014092a834c2496c3e3/src/mono/mono/metadata/icall-def-netcore.h#L177 https://github.com/dotnet/runtime/blob/2a1735b162437bb35a112014092a834c2496c3e3/src/mono/mono/metadata/icall-decl.h#L142 https://github.com/dotnet/runtime/blob/2a1735b162437bb35a112014092a834c2496c3e3/src/mono/mono/metadata/icall-decl.h#L146

Reply:
@am11 `icall-decl.h` should be preserved, it's used by 6.x mono

Reply:
Ah, ok I see. We can then just delete two lines in `icall-def-netcore.h` and leave `icall-decl.h` and `sysmath.c`. Updated my comment above.